dojo/forms.py

```diff
@@ -401,10 +401,11 @@ class ImportScanForm(forms.Form):
                          ("AWS Security Hub Scan", "AWS Security Hub Scan"),
                          ("GitLab SAST Report", "GitLab SAST Report"),
                          ("HuskyCI Report", "HuskyCI Report"),
+                         ("Semgrep JSON Report", "Semgrep JSON Report"),
                          ("Risk Recon API Importer", "Risk Recon API Importer"),
-                         ("DrHeader JSON Importer", "DrHeader JSON Importer"),
```

Comment:
Was this meant to be removed?

Reply:
I think this was a mistake
DrHeader is already imported

Reply:
looks ok here